Makefile

```diff
@@ -4,7 +4,8 @@ REPO=composablefi
 SERVICE_NAME=composable
 INSTALL_DIR=docker/
 IMAGE_URL:=${REPO}/${SERVICE_NAME}
-CARGO_VERSION:=$(shell git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1 )
+RELEASE_VERSION:=$(shell git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1)
```

Comment:
```suggestion
```

Reply:
We don't need to remove this...the way the docker release it's set, the release version needs to be set in the makefile and it doesnt affect the way release-pipeline works